lua/plugins/filetype.lua

```diff
@@ -46,7 +46,7 @@ vis.ftdetect.filetypes = {
 	},
 	bash = {
 		utility = { "^[db]ash$", "^sh$","^t?csh$","^zsh$" },
-		ext = { "%.bash$", "%.csh$", "%.sh$", "%.zsh$" ,"^APKBUILD$", "%.ebuild$", "^.bashrc$", "^.bash_profile$" },
+		ext = { "%.bash$", "%.csh$", "%.sh$", "%.zsh$" ,"^APKBUILD$", "%.ebuild$", "^.bashrc$", "^.bash_profile$", "^.yashrc$", "^.yash_profile$" },
```

Comment:
i would argue to change this to `^.*sh_profile$`, vice versa for .yashrc; and add `^.profile$`